pallets/registry/src/mock.rs

```diff
@@ -175,9 +176,20 @@ impl pallet_balances::Config for MockRuntime {
 	type ReserveIdentifier = ();
 }
 
+pub struct AuthorGiven;
+
+impl FindAuthor<u64> for AuthorGiven {
+	fn find_author<'a, I>(_digests: I) -> Option<u64>
+	where
+		I: 'a + IntoIterator<Item = (ConsensusEngineId, &'a [u8])>,
+	{
+		Some(100)
+	}
+}
+
```

Comment:
Does anybody know what was the issue with that? Only saw in the commits of the latest version, that they changed the import logic of the node.

Reply:
Storage of the block author `author()` is now an `OptionQuery` instead of a `ValueQuery` over the AccountId, which the trait now it doesn't support a Default value. Is either set or not. Hence this cascading list of changes.